app/src/main/java/com/github/wnder/GuessLocationActivity.java

```diff
@@ -370,7 +376,7 @@ private void compassModeSetup(){
             sensorManager.registerListener(listener, (Sensor) list.get(0), SensorManager.SENSOR_DELAY_NORMAL);
             updateGuessPositionFromGPS.run();
             mapboxMap.getUiSettings().setRotateGesturesEnabled(false);
-            compassModeButtonView.setText(R.string.switchCompassModeText);
+            compassModeButtonView.setForeground(getDrawable(R.drawable.ic_outline_explore_24));
```

Comment:
Using FloatingActionButton you can the use setImageResource that should do what you want here.

Reply:
Well I tried to do that but I couldn't change the size of the icon properly and change it in white color with that function. For now I would like to let it that way. And maybe someone will have to double check the whole app UI latter to homogenize better.

Reply:
I think it's fine then, the important thing here is the look I guess.